### PR TITLE
Fix IsPassingResult() and Coalesce().

### PR DESCRIPTION
**Pull request by fejta**
BUILD_PASSED should be a passing result.
Coalesce should consider anything not passing (and not a special case) a failure.

/assign @michelle192837

### CODE REVIEW

**Comment by fejta** [comment]
Okay, finally got this one passing

File: internal/result/results.go [hunk 1]
```diff
@@ -51,26 +51,26 @@ var (
...
 
-// LTE returns if rowResult is less than or equal to compareTo.
-func LTE(rowResult, compareTo statuspb.TestStatus) bool {
+// lte returns if rowResult is less than or equal to compareTo.
+func lte(rowResult, compareTo statuspb.TestStatus) bool {
 	return statusSeverity[rowResult] <= statusSeverity[compareTo]
 }
 
-// GTE returns if rowResult is greater than or equal to compareTo.
-func GTE(rowResult, compareTo statuspb.TestStatus) bool {
```
Comment (michelle192837) [inline internal/result/results.go:60]: Can we keep this exported for internal use, or should we just have it exported there?

**Review by michelle192837 — APPROVED** [review]
/hold

Unhold after comment addressed!

File: internal/result/results.go (hunk 1 above)
Comment (fejta) [reply internal/result/results.go:60]: It wasn't exported historically? Does something need to use it internally?

**Comment by fejta** [comment]
/hold cancel

Will send a subsequent PR if we need this exported

**Pull request merged**